src/test/scala/io/iohk/ethereum/vm/CallOpcodesSpec.scala

```diff
@@ -138,6 +138,18 @@ class CallOpcodesSpec extends WordSpec with Matchers {
         call.stateOut.stack.pop._1 shouldEqual UInt256.One
       }
 
+      "should store contract's return data in memory" in {
+        //here the passed data size is equal to the contract's return data size
+
+        val expectedData = fxt.inputData.take(fxt.inputData.size / 2)
```

Comment:
minor detail. Can we explain with a short comment where this `2` comes from? I think it's related with the code being called but...